### PR TITLE
Login create reset change profile 2

### PR DESCRIPTION
**Pull request by jg6604**

### CODE REVIEW

**Comment by madarez** [comment]
@jg6604 , are you available to join me on Slack to pair program on this? I don't have a way of reaching you...

File: app/create-acc/page.js [hunk 1]
```diff
@@ -1,221 +1,167 @@
...
-        // Redirect to profile page or any other page as needed
-        router.push('/profile'); 
-    } catch (error) {
-        alert(error.message);
+            // Create a document in Firestore in "users" collection with UID as the document key
+            await setDoc(doc(db, "users", uid), {
+                created_at: new Date(),
+                first_name,
+                last_name,
+                email:user.email,
```
Comment (madarez) [inline app/create-acc/page.js:60]: No email is to be stored on the user object. The users object only uses the UID as the ID.
```suggestion
```

File: app/create-acc/page.js [hunk 2]
```diff
@@ -1,221 +1,167 @@
...
+                            <input id="last-name" type="text" placeholder="Ex: Doe" style={{ border: '0px', borderBottom: '1px solid black', padding: '10px', width: '100%', margin: '0 auto', display:'block', color: 'black' }} required className="appearance-none rounded-none relative block w-full px-3 py-2 border border-gray-300 placeholder-gray-500 text-gray-900 rounded-t-md focus:outline-none focus:ring-blue-500 focus:border-blue-500 focus:z-10 sm:text-sm" value={last_name} onChange={(e) => setLastName(e.target.value)} />
+                        </div>
+                    </div>
+                    <div>
+                        <label htmlFor="email" className="text-sm font-medium text-gray-700 block mb-2">Email</label>
+                        <input id="email"  style={{ border: '0px', borderBottom: '1px solid black', padding: '10px', width: '100%', margin: '0 auto', display:'block', color: 'black' }} required className="appearance-none rounded-none relative block w-full px-3 py-2 placeholder-gray-500 text-gray-900 focus:outline-none focus:ring-blue-500 focus:border-blue-500 sm:text-sm" value={user.email}/>
```
Comment (madarez) [inline app/create-acc/page.js:105]: Not accessing it like `user?.email` is causing the compilation error. Please consider reading `currentUser`'s attributes as explain [here](https://firebase.google.com/docs/auth/web/manage-users#get_a_users_profile).

Comment (jg6604) [reply app/create-acc/page.js]: Thank you.

**Comment by jg6604** [comment]
It seems it was deployed successfully.

**Pull request closed without merging**